src/components/molecules/Schedule/Schedule.types.ts

```diff
@@ -0,0 +1,19 @@
+import { Room } from "types/rooms";
+import { PersonalizedVenueEvent } from "types/venues";
+import { WithVenueId } from "utils/id";
+
+export type RoomWithEvents = Room & {
+  events: WithVenueId<PersonalizedVenueEvent>[];
+};
+
+export interface ScheduleDay {
+  isToday: boolean;
+  weekday: string;
+  dayStartUtcSeconds: number;
+  rooms: RoomWithEvents[];
+  personalEvents: WithVenueId<PersonalizedVenueEvent>[];
+}
+
+export interface ScheduleProps {
+  scheduleDay: ScheduleDay;
+}
```

Comment:
should be moved to the component

Reply:
Further context: This `Foo.types.ts` pattern is something we want to eliminate from the codebase, that was implemented by a former engineer against our patterns/standards.

Reply:
moved props to Schedule.tsx https://github.com/sparkletown/sparkle/pull/1302/commits/97450fe8a6c8cff5135e9d09b572068adf69b71c